include/openmc/distribution_spatial.h

```diff
@@ -136,6 +136,34 @@ class MeshSpatial : public SpatialDistribution {
                                 //!< mesh element indices
 };
 
+//==============================================================================
+//! Distribution of points
+//==============================================================================
+
+class PointCloud : public SpatialDistribution {
+public:
+  explicit PointCloud(pugi::xml_node node);
+  explicit PointCloud(gsl::span<const double> x, gsl::span<const double> y,
```

Comment:
Note: this could just as easily be a vector of `Position`s but I wasn't sure how to do that in XML space, so decided to just keep it this way for now.  Happy for any guidance on changing this to a different form.

Could also overload the constructor with a few choices, but not sure of OpenMC design practices.

Reply:
It looks like adding an input stream operator for `Position` will allow existing code to work, although not sure about robustness